### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by tiserett**
[DEMO LINK](https://tiserett.github.io/layout_creativeBakery/).

I tried to implement it as good as I can, it's not pixel perfect but it’s honest work.

### CODE REVIEW

**Review by AlinaMatuschak — CHANGES_REQUESTED** [review]
- add hovers
Screenshot 2022-08-01 at 16 16 34

- "learn more" btn does nothing. Add scroll to some block 
- check spaces between blocks
- you can centralize vertically icon with text (looks broken)
Screenshot 2022-08-01 at 16 28 13

- I can’t see all mobile menu, Scroll doesn’t work, and I don’t see all links
Screenshot 2022-08-01 at 16 56 53

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,283 @@
...
+        </div>
+      </section>
+
+      <section class="products" id="products">
+        <div class="container">
+          <h2 class="products__title">What we bake</h2>
+          <p class="products__below-title">We can customize our cookies right for your needs. Just imagine it</p>
+
+          <section class="products__container">
+            <article class="card card-1">
```
Comment (AlinaMatuschak) [inline src/index.html:89]: nth-child

```suggestion
            <article class="card">
```

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,283 @@
...
+              <p class="card__description">Chocolate biscuit coffee cake topped with a buttercream and powder</p>
+              <a href="#" class="card__order">Order</a>
+            </article>
+
+            <article class="card card-3"
+          >
+              <img
+                src="./images/content/product3.png"
+                alt="cake"
+                class="card__img">
```
Comment (AlinaMatuschak) [inline src/index.html:116]:
```suggestion
            <article class="card card-3">
              <img
                src="./images/content/product3.png"
                alt="cake"
                class="card__img"
              >
```

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,283 @@
...
+
+      <section class="about" id="about-us">
+        <div class="container">
+          <article class="about__content">
+            <h2 class="about__title">Few words About us</h2>
+            <p class="about__text">
+              Like many design teams, Onfido made the transition from Abstract + Sketch, to Figma in the last couple of years. One of the common resistance points internally before making that switch, was Figma`s lack of Git-style branching that Abstract enabled. The design team had come to rely on branching and had built a lot of our processes around it.
+            </p>
+
+            <article class="about__info about__info-1">
```
Comment (AlinaMatuschak) [inline src/index.html:166]:
```suggestion
            <article class="about__info">
```

File: src/styles/blocks/footer.scss [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    @include onTablet {
+      grid-area: 4 / 1 / 5 / 2;
+    }
+
+    @include onLaptop {
+      grid-area: 4 / 1 / 5 / 3;
+    }
+
+    &-link {
+      transition: all 0.3s;
```
Comment (AlinaMatuschak) [inline src/styles/blocks/footer.scss:124]: Always add transition property. Hard for the browser to control all properties

Comment (AlinaMatuschak) [inline src/styles/blocks/footer.scss:124]: you can create variable for 0.3s

File: src/styles/blocks/main/products.scss [hunk 5]
```diff
@@ -0,0 +1,28 @@
+.products {
+  margin-bottom: 80px;
```
Comment (AlinaMatuschak) [inline src/styles/blocks/main/products.scss:2]: Margin for bem block?

File: src/styles/blocks/page.scss [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+p,
+h1,
+h2,
+ul,
+a {
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  list-style: none;
+}
```
Comment (AlinaMatuschak) [inline src/styles/blocks/page.scss:14]: It shouldn’t be on the page file. Create normalize and reset

**Review by AlinaMatuschak — APPROVED** [review]
GJ 🔥

- add hovers
Screenshot 2022-08-02 at 17 39 57

File: src/index.html (hunk 3 above)
Comment (tiserett) [reply src/index.html:166]: I can't rewrite it with nth-child, so I'll leave it like this, but I fixed cards. idk why it's not working ;(

**Pull request closed without merging**